docs/release-notes/self-hosted/self-hosted-v7.1.0.md

```diff
@@ -0,0 +1,101 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v7.1.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/5.5.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.7.38
+---
+
+# Self-hosted v7.1.0
+
+These release notes are for [Codacy Self-hosted v7.1.0](https://github.com/codacy/chart/releases/tag/7.1.0){: target="_blank"}, released on May 11, 2022. <!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues without release notes
+
+Others:
+-   https://codacy.atlassian.net/browse/CY-6021
+-   https://codacy.atlassian.net/browse/CY-5555
+
+Jira issues with disabled release notes
+-   https://codacy.atlassian.net/browse/CY-5932
```

Comment:
Check if the changes related to [CY-5932](https://codacy.atlassian.net/browse/CY-5932) are included in this release.